doc/instruction_decode_execute.rst

```diff
@@ -165,3 +162,13 @@ Source File: :file:`rtl/ibex_load_store_unit.sv`
 
 The Load-Store Unit (LSU) interfaces with main memory to perform load and store operations.
 See :ref:`load-store-unit` for more details.
+
+.. rubric:: Footnotes
+
+.. [#B_draft] Ibex fully implements draft version 0.92 of the RISC-V Bit Manipulation Extension.
```

Comment:
"implements the"

Reply:
I don't think that's right. It implements version N of the spec (not "the version N of the spec").

Reply:
Left as is.